src/cmd/ksh93/tests/shtests

```diff
@@ -323,6 +323,14 @@ do	if	[[ ! -r $i ]]
 		[[ $utf8 == 0 || $i == $setslocale ]] || locales+=" C.UTF-8"
 		for lang in $locales
 		do	o=$u
+
+			# This test only fails on openSUSE on Travis
+			if [[ $i == "builtins.sh" ]] && [[ $lang == "C.UTF-8" ]] && [[ $IS_RUNNING_TRAVIS ]] && $(grep -q openSUSE /etc/os-release)
```

Comment:
Same thing here: the `$( )` form seems wrong since we want to test the exit status, not whether `grep -q` wrote anything to stdout (which it wouldn't).